gf2.cpp

```diff
@@ -1003,6 +1016,22 @@ void CommandDeleteBreakpoint(void *_index) {
 	DebuggerSend(buffer, true, false);
 }
 
+void CommandDisableBreakpoint(void *_index) {
+	int index = (int) (intptr_t) _index;
+	Breakpoint *breakpoint = &breakpoints[index];
+	char buffer[1024];
+	StringFormat(buffer, 1024, "disable %d", breakpoint->number);
+	DebuggerSend(buffer, true, false);
+}
+
+void CommandEnableBreakpoint(void *_index) {
+	int index = (int) (intptr_t) _index;
+	Breakpoint *breakpoint = &breakpoints[index];
+	char buffer[1024];
+	StringFormat(buffer, 1024, "enable %d", breakpoint->number);
+	DebuggerSend(buffer, true, false);
+}
```

Comment:
We could merge `CommandDisableBreakpoint()` and `CommandEnableBreakpoint()` to `CommandToggleDisableBreakpoint()`, but i don't know if keeping them separate could come in handy in the futur?